3rd-party/web-stories.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Filter to enable web stories built in open graph data from being output.
+ *
+ * @param bool $enabled If web stories open graph data is enabled.
+ *
+ * @return bool
+ */
+function web_stories_disable_open_graph( $enabled ) {
```

Comment:
Only the Web Stories plugin should be using the `web_stories_` prefix for functions

Reply:
Fixed